dashboard/src/components/Catalog/CatalogItems.tsx

```diff
@@ -9,9 +9,16 @@ interface ICatalogItemsProps {
   csvs: IClusterServiceVersion[];
   cluster: string;
   namespace: string;
+  hasFinished: boolean;
 }
 
-export default function CatalogItems({ charts, csvs, cluster, namespace }: ICatalogItemsProps) {
+export default function CatalogItems({
+  charts,
+  csvs,
+  cluster,
+  namespace,
+  hasFinished,
```

Comment:
`hasFinished` isn't really descriptive. I assume this means `hasFinishedFetching`?